scripts/test_antlr_grammar.sh

```diff
@@ -2,11 +2,10 @@
 
 set -e
 
-ROOT_DIR="$(dirname "$0")"/..
+ROOT_DIR=$(readlink -f "$(dirname "$0")"/..)
```

Comment:
I thought we wanted to get rid of `readlink`? ;)

Reply:
The script uses it anyways and I needed it... not sure why :-). We can get rid of it separately :-).